pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -103,6 +107,7 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
     event AllowlistAddressRemoved(address indexed member);
     event TokenAdded(IERC20 indexed token, uint256 normalizedWeight);
     event TokenRemoved(IERC20 indexed token);
+    event CircuitBreakerSet(IERC20 indexed token, uint256 lowerBound, uint256 upperBound);
```

Comment:
See [this review comment](https://github.com/balancer-labs/balancer-v2-monorepo/pull/1792#discussion_r974576492)